free-2.md

```diff
@@ -58,12 +101,21 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think getting nice-looking output would be a lot of non-language work.
+But a proof of concept would be almost entirely language and design work, because
+it would essentially be trying to redesign LaTeX as minimalistically as possible.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+This is a pretty ambitious project for it to be usable, but a proof of concept that
+has bad graphics could be manageable. It would probably produce HTML like Markdown does.
```

Comment:
Producing HTML would run counter to LaTeX's "your document will look the same no matter who compiles it" philosophy, which I like. It's your choice though.